src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -164,7 +166,7 @@ def assertCbsdRecord(self, registration_request, grant_request, grant_response,
               self.assertFalse('groupingParam' in cbsd_record[0])
           
           # Get grants by cbsd_id
-          grants_of_cbsd = [cbsd['grants'] for cbsd in cbsd_dump_data if cbsd['id'] == record_id]
+          grants_of_cbsd = [cbsd['grants'] for cbsd in cbsd_dump_data if cbsd['id'] == record_id][0]
```

Comment:
I don't think you can do this directly, as it invalidates the following line.

Reply:
actually  cbsd['grants'] is an array.
[cbsd['grants'] for cbsd in cbsd_dump_data if cbsd['id'] == record_id] is an array of array, so with taking the first element with is an array, the following lines still valid.

Reply:
Hmm, good point. But aren't you still making the assumption that there is only one CBSD with that ID? I guess it should be guaranteed by line 84. I'd still feel more comfortable with a check (or at least a comment) here, but I will leave the decision to you since you are the author :)